libs/documentation/src/lib/storybook/formly/formly-filter/formly-filter-horizontal/formly-filter-horizontal.component.html

```diff
@@ -7,13 +7,9 @@ <h4 class="sds-card__title display-block margin-bottom-2">Horizontal Filters</h4
       [model]="model"
       [horizontal]="true"
       (filterChange)="onFilterChange($event)"
+      [enableSearchfield]="true"
```

Comment:
Do we always want there to be a search field with horizontal filters? When you brought this change up in FWG it sounded like one of the teams didn't have a search. But maybe I misunderstood what he was saying.

Reply:
You are right we always don’t have search in horizontal filters, added another input ‘enableSearchField’ to accommodate the feature

Reply:
Yeah you do. For some reason I was thinking this is a component, not a demo. Looks good.